test/BaseTest.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {UniversalRewardsDistributor} from "src/UniversalRewardsDistributor.sol";
+
+import "@forge-std/Test.sol";
+
+abstract contract BaseTest is Test {
+    function _precomputeAddress(address sender, bytes memory encodedParams, bytes32 salt)
```

Comment:
Should this function be part of the factory? Should be addressed in a dedicated issue